src/main/java/seedu/duke/userinterface/Mode.java

```diff
@@ -1,6 +1,56 @@
 package seedu.duke.userinterface;
 
-public enum Mode {
-    NOTEBOOK,
-    TIMETABLE
+import seedu.duke.notebooks.Notebook;
+import seedu.duke.notebooks.NotebookShelf;
+import seedu.duke.notebooks.Section;
+import seedu.duke.tasks.TaskList;
+
+public class Mode {
```

Comment:
Do you think something like `ApplicationState` would be a more appropriate name, since this class encapsulates more than just the mode?